packages/urls/src/urls.ts

```diff
@@ -1,167 +1,296 @@
+import { withPlatformUtm } from './platform-utm';
 import { Url } from './types';
 
+// =====================
+// 🧩 TREZOR SUITE - CORE
+// =====================
+
+export const SUITE_WEB_URL = 'https://suite.trezor.io/web/';
+export const SUITE_URL: Url = 'https://trezor.io/trezor-suite';
+
+export const SUITE_BACKUP_URL: Url = 'https://suite.trezor.io/web/backup/';
+export const SUITE_FIRMWARE_URL: Url = 'https://suite.trezor.io/web/firmware/';
+export const SUITE_UDEV_URL: Url = 'https://suite.trezor.io/web/udev/';
+export const SUITE_WEB_DEVICE_SETTINGS_URL = (SUITE_WEB_URL + 'settings/device/') as Url;
+
 export const TREZOR_URL: Url = 'https://trezor.io';
 export const TREZOR_FORUM_URL: Url = 'https://forum.trezor.io/';
 export const TREZOR_START_URL: Url = 'https://trezor.io/start';
-export const TREZOR_SUPPORT_URL: Url = 'https://trezor.io/support';
 export const TREZOR_RESELLERS_URL: Url = 'https://trezor.io/resellers/';
 export const TREZOR_COINS_URL: Url = 'https://trezor.io/coins';
```

Comment:
Why is `withPlatformUtm` not used for those URLs? I don't know if it's needed, but it would be nice to have it confirmed.

Reply:
@fang314 could you validate the urls mentioned in the comments by @matejkriz pls? I was not sure if we should use it for them as well and forgot to ask 🙇‍♂️